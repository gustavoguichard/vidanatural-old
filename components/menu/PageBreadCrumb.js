import { memo } from 'react'

const PageBreadCrumb = ({ title, ...props }) => (
  <div className="page-breadcrumb" {...props}>
    <img src="/static/page-feather.png" alt="icone" />
    <span>{title}</span>
  </div>
)

export default memo(PageBreadCrumb)
