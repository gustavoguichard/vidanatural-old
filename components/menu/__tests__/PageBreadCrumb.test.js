import { render } from 'react-testing-library'
import PageBreadCrumb from '../PageBreadCrumb'

it('renders correctly', () => {
  const { container } = render(<PageBreadCrumb title="Foo" bar="zaz" />)
  expect(container).toMatchSnapshot()
})
