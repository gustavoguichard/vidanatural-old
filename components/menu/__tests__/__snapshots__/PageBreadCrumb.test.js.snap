// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    bar="zaz"
    class="page-breadcrumb"
  >
    <img
      alt="icone"
      src="/static/page-feather.png"
    />
    <span>
      Foo
    </span>
  </div>
</div>
`;
